modules/ROOT/pages/asset-types.adoc

```diff
@@ -48,6 +48,17 @@ Configuration modules to extend the functionality of an API and enforce capabili
 * REST APIs
 +
 RAML or OAS files that specify APIs referenced by an HTTP Request connector to expose metadata to Anypoint Studio.
+
+* RPA Templates
++
+** RPA Activity Templates
++
+Activity templates are reusable code within RPA to standardize the most used activity steps and make them available to other developers in the organization. File type is .calw.
```

Comment:
```suggestion
Activity templates are reusable code within RPA to standardize the most used activity steps and make them available to other developers in the organization. File type is `.calw`.
```

Reply:
(global, unless it's a huge effort)

Reply:
SR doesn't really cover this exactly.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-filetypesandfileextensions

Reply:
fixed